### PR TITLE
tidy up build system

### PR DESCRIPTION
**Pull request by teknoraver**
Remove files generated by autotools and add them in .gitignore, such files should be shipped only in tagged release and not in the master tree.
Add binary and intermediate outpuf files in ignore as well

### CODE REVIEW

**Comment by IradaPalgawong** [comment]
bj

File: netperf.spec.in [hunk 1]
```diff
@@ -7,7 +7,7 @@ Group:         System Environment/Base
 License:       Unknown
 URL:           http://www.netperf.org/
 Packager:      Martin A. Brown
-Source:        ftp://ftp.netperf.org/netperf/%{name}-%{version}.tar.bz2
+Source:        https://github.com/HewlettPackard/%{name}/archive/master.tar.gz
```
Comment (grundlerchromium) [inline netperf.spec.in]: Matteo, are you sure building "master" is the right thing to do for all redhat based distros?
Or is the %{version} branch just missing from github?

Every distro I've ever used picked up the version (or "release branch"), not the development branch.

Comment (grundlerchromium) [reply netperf.spec.in]: Hrm...it looks like https://github.com/HewlettPackard/netperf/archive/netperf-2.7.0.tar.gz exists... so I'm curious why "master" is the preferred tarball here. TIA!

**Comment by grundlerchromium** [comment]
BTW - This LGTM regardless of the specfile change.

File: netperf.spec.in (hunk 1 above)
Comment (teknoraver) [reply netperf.spec.in]: Yes, usualy a tagged release should be used, but this is the master branch and refers to master, in other tags it should point to %{version}

Comment (grundlerchromium) [reply netperf.spec.in]: Ah! Yeah, that makes sense - thanks!

**Pull request merged**